codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Frame.java

```diff
@@ -19,4 +19,11 @@
 
 /** An HTTP/2 frame. */
 @UnstableApi
-public interface Http2Frame { }
+public interface Http2Frame {
+
+    /**
+     * Returns the name of the HTTP/2 frame e.g. DATA, GOAWAY, etc.
+     */
+    String name();
```

Comment:
I am using it in exception messages. I wanted to mimic the messages of the existing HTTP/2 API, where they use DATA, GOAWAY over class names.